packages/remix-node/globals.ts

```diff
@@ -35,6 +36,8 @@ declare global {
 
       ReadableStream: typeof ReadableStream;
       WritableStream: typeof WritableStream;
+
+      AbortController: typeof AbortController;
```

Comment:
@jacob-ebey Looking at the `abort-controller` repo, it looks like there may be a few issues with its typings that could potentially cause issues when passing signals. 

https://github.com/mysticatea/abort-controller/issues/16#issuecomment-592464104
https://github.com/mysticatea/abort-controller/issues/36

Reply:
It's not an issue at runtime as we are currently running tests using it. As long as the build passes and doesn't have type issues it's not an issue.